Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN export LIQUIBASE_HOME=/liquibase
 # Install Drivers
 RUN lpm update
 RUN lpm add snowflake --global
+RUN lpm add liquibase-snowflake --global
```

Comment:
I recommend this become one line with `lpm add snowflake liquibase-snowflake --global`. `lpm` can install multiple packages at one time.

Reply:
Lines 47 and 48 can be combine is what I'm trying to say.

Reply:
I will submit another pull request for this change.